run.sh

```diff
@@ -130,7 +130,7 @@ deploy(){
     curl --silent --fail ${K3D_URL} | TAG=${K3D_VERSION} bash
 
     echo -e "\existing_network${YELLOW}Deploy cluster ${CYAN}$name ${NC}"
-    eval "k3d cluster create $name --wait $arguments --network $network $registryArg"
+    eval "k3d cluster create $name --wait $arguments --network $network $registryArg" --verbose
```

Comment:
Do we need to keep verbose?

Reply:
I'd probably make it a configurable parameter.

Reply:
@somaritane , good idea 👍   we can run already run `--verbose` an argument.

see usage in temporary k8gb workflow : https://github.com/k8gb-io/k8gb/blob/upgrade_to_k3d-action_v1.5.0/.github/workflows/terratest.yaml#L27